### PR TITLE
Activate/Deactivate Conference Calling Team Feature for Back Office

### PR DESCRIPTION
**Pull request by battermann**
## Checklist

 - [ ] The **PR Title** explains the impact of the change.
 - [ ] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [ ] If HTTP endpoint paths have been added or renamed, the **endpoint / config-flag checklist** (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
 - [ ] If a cassandra schema migration has been added, I ran **`make git-add-cassandra-schema`** to update the cassandra schema documentation.
 - [ ] **changelog.d** contains the following bits of information ([details](https://github.com/wireapp/wire-server/blob/develop/docs/developer/changelog.md)):
   - [ ] A file with the changelog entry in one or more suitable sub-sections. The sub-sections are marked by directories inside `changelog.d`.
   - [ ] If new config options introduced: added usage description under docs/reference/config-options.md
   - [ ] If new config options introduced: recommended measures to be taken by on-premise instance operators.
   - [ ] If a cassandra schema migration is backwards incompatible (see also [these docs](https://github.com/wireapp/wire-server/blob/develop/docs/developer/cassandra-interaction.md#cassandra-schema-migrations)), measures to be taken by on-premise instance operators are explained.
   - [ ] If a data migration (not schema migration) introduced: measures to be taken by on-premise instance operators.
   - [ ] If public end-points have been changed or added: does nginz need un upgrade?
   - [ ] If internal end-points have been added or changed: which services have to be deployed in a specific order?

### CODE REVIEW

File: services/galley/schema/src/V56_ConferenceCallingPaymentStatus.hs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 56 "Add payment status config for conference calling" $ do
```
Comment (jschaul) [inline services/galley/schema/src/V56_ConferenceCallingPaymentStatus.hs:28]: Please combine the statements of 55 and 56 into a single migration step, if introduced in one and the same PR. That can be done in the same CQL statement as here you're operating on the same table; or in general also in multiple CQL statements but of the same migration, see e.g. https://github.com/wireapp/wire-server/blob/develop/services/brig/schema/src/V30.hs.

Comment (fisx) [reply services/galley/schema/src/V56_ConferenceCallingPaymentStatus.hs:28]: It's two different PRs, but this one is based on #1937, which hasn't been merged yet.

**Comment by fisx** [comment]
I've changed the base of the PR, we can change it back when #1937 has landed on develop.  The merge conflicts are going to be fun, but I'll take care of that.

**Comment by fisx** [comment]
@battermann please take a look at https://github.com/wireapp/wire-server/pull/1943, and close this in favor of the new one if you think I've resolved the conflicts correctly.

**Comment by fisx** [comment]
@battermann can this be closed in favor of https://github.com/wireapp/wire-server/pull/1943?

**Comment by battermann** [comment]
@fisx I think this can be closed in favor of #1943

**Pull request closed without merging**